Jun 26, 2021 10:35:33 PM hudson.remoting.jnlp.Main createEngine
INFO: Setting up agent: agent 1
Jun 26, 2021 10:35:33 PM hudson.remoting.jnlp.Main$CuiListener <init>
INFO: Jenkins agent is running in headless mode.
Jun 26, 2021 10:35:33 PM hudson.remoting.Engine startEngine
INFO: Using Remoting version: 4.5
Jun 26, 2021 10:35:33 PM org.jenkinsci.remoting.engine.WorkDirManager initializeWorkDir
INFO: Using /home/harrington/designProject2020/automation/Automation/remoting as a remoting work directory
Jun 26, 2021 10:35:34 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Locating server among [http://localhost:8080/]
Jun 26, 2021 10:35:34 PM org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver resolve
INFO: Remoting server accepts the following protocols: [JNLP4-connect, Ping]
Jun 26, 2021 10:35:34 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Agent discovery successful
  Agent address: localhost
  Agent port:    33645
  Identity:      90:62:16:69:24:e3:fb:b3:4c:4c:b7:66:93:c9:c3:9a
Jun 26, 2021 10:35:34 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Handshaking
Jun 26, 2021 10:35:34 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Connecting to localhost:33645
Jun 26, 2021 10:35:34 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Trying protocol: JNLP4-connect
Jun 26, 2021 10:35:35 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Remote identity confirmed: 90:62:16:69:24:e3:fb:b3:4c:4c:b7:66:93:c9:c3:9a
Jun 26, 2021 10:35:35 PM org.jenkinsci.remoting.protocol.impl.ConnectionHeadersFilterLayer onRecv
INFO: [JNLP4-connect connection to localhost/127.0.0.1:33645] Local headers refused by remote: agent 1 is already connected to this master. Rejecting this connection.
Jun 26, 2021 10:35:35 PM hudson.remoting.jnlp.Main$CuiListener status
INFO: Protocol JNLP4-connect encountered an unexpected exception
java.util.concurrent.ExecutionException: org.jenkinsci.remoting.protocol.impl.ConnectionRefusalException: agent 1 is already connected to this master. Rejecting this connection.
	at org.jenkinsci.remoting.util.SettableFuture.get(SettableFuture.java:223)
	at hudson.remoting.Engine.innerRun(Engine.java:744)
	at hudson.remoting.Engine.run(Engine.java:519)
Caused by: org.jenkinsci.remoting.protocol.impl.ConnectionRefusalException: agent 1 is already connected to this master. Rejecting this connection.
	at org.jenkinsci.remoting.protocol.impl.ConnectionHeadersFilterLayer.newAbortCause(ConnectionHeadersFilterLayer.java:378)
	at org.jenkinsci.remoting.protocol.impl.ConnectionHeadersFilterLayer.onRecvClosed(ConnectionHeadersFilterLayer.java:433)
	at org.jenkinsci.remoting.protocol.ProtocolStack$Ptr.onRecvClosed(ProtocolStack.java:816)
	at org.jenkinsci.remoting.protocol.FilterLayer.onRecvClosed(FilterLayer.java:288)
	at org.jenkinsci.remoting.protocol.impl.SSLEngineFilterLayer.onRecvClosed(SSLEngineFilterLayer.java:172)
	at org.jenkinsci.remoting.protocol.ProtocolStack$Ptr.onRecvClosed(ProtocolStack.java:816)
	at org.jenkinsci.remoting.protocol.NetworkLayer.onRecvClosed(NetworkLayer.java:154)
	at org.jenkinsci.remoting.protocol.impl.BIONetworkLayer.access$1500(BIONetworkLayer.java:48)
	at org.jenkinsci.remoting.protocol.impl.BIONetworkLayer$Reader.run(BIONetworkLayer.java:247)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:118)
	at java.lang.Thread.run(Thread.java:748)
	Suppressed: java.nio.channels.ClosedChannelException
		... 7 more

Jun 26, 2021 10:35:35 PM hudson.remoting.jnlp.Main$CuiListener error
SEVERE: The server rejected the connection: None of the protocols were accepted
java.lang.Exception: The server rejected the connection: None of the protocols were accepted
	at hudson.remoting.Engine.onConnectionRejected(Engine.java:829)
	at hudson.remoting.Engine.innerRun(Engine.java:769)
	at hudson.remoting.Engine.run(Engine.java:519)

